src/interface/TransactionRequestsService-swagger.yaml

```diff
@@ -986,10 +987,15 @@ components:
     AuthenticationValue:
       title: AuthenticationValue
```

Comment:
authenticationValue could be a string (OTP & QRCODE) or object (U2FPinValue) - I already have this definition in my work done

Reply:
Hmm https://github.com/mojaloop/pisp/blob/master/docs/fspiop-rest-v1.0-OpenAPI-implementation_openapi3.yaml#L2014 yes?

Unfortunately hapi doesn't support OA3 and it's `oneOf` field. So I am not sure what to do.

I had based my changes on https://github.com/mojaloop/pisp/blob/master/docs/fspiop-rest-v1.0-OpenAPI-implementation.yaml#L3663

I didn't notice that spec files were different.

Reply:
@kleyow take a look: https://github.com/mojaloop/transaction-requests-service/pull/47